-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-753264/25645/summary.html
**COMMIT**: 0d151a0b0962baf94ab5b1f5b314c3282150c2e6
**CMSSW**: CMSSW_12_5_X_2022-06-20-1100/el8_amd64_gcc10
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/38437/25645/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>
>> Plugins of all types refreshed.
gmake[1]: *** [config/SCRAM/GMake/Makefile.rules:1826: CompilePython] Error 1
gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/pool/condor/dir_155243/jenkins/workspace/ib-run-pr-tests/CMSSW_12_5_X_2022-06-20-1100'
gmake: *** [config/SCRAM/GMake/Makefile.rules:1714: src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
+ eval scram build outputlog '&&' '(python3' /pool/condor/dir_155243/jenkins/workspace/ib-run-pr-tests/cms-bot/buildLogAnalyzer.py --ignoreWarning=Wdeprecated-declarations --logDir /pool/condor/dir_155243/jenkins/workspace/ib-run-pr-tests/CMSSW_12_5_X_2022-06-20-1100/tmp/el8_amd64_gcc10/cache/log/src '||' 'true)'
++ scram build outputlog
>> Entering Package Configuration/Eras
</pre>

--- a/RecoTracker/IterativeTracking/python/PixelLessStep_cff.py
+++ b/RecoTracker/IterativeTracking/python/PixelLessStep_cff.py
@@ -398,7 +398,7 @@ trackdnn.toReplaceWith(pixelLessStep, trackTfClassifier.clone(
 ))
 (trackdnn & fastSim).toModify(pixelLessStep,vertices = 'firstStepPrimaryVerticesBeforeMixing')
 
-((~trackingMkFitPixelLessStep) & trackdnn).toModify(pixelLessStep, mva.tfDnnLabel = 'trackSelectionTf_CKF',
+((~trackingMkFitPixelLessStep) & trackdnn).toModify(pixelLessStep, mva = dict(tfDnnLabel  = 'trackSelectionTf_CKF'),
                                                     qualityCuts = [-0.81, -0.61, -0.17])
 
 pp_on_AA.toModify(pixelLessStep, qualityCuts = [-0.4,0.0,0.8])

